addons/website_livechat/models/discuss_channel.py

```diff
@@ -51,7 +39,14 @@ def _to_store_defaults(self, for_current_user=True):
                 predicate=lambda channel: channel.channel_type == "livechat"
                 and self.livechat_visitor_id.has_access("read"),
             ),
-            "requested_by_operator",
+            # sudo: discuss.channel - visitor can access channel member history
```

Comment:
This is not exactly the right explanation. It's not allowed generally speaking.

It's only allowed when it's accessing the history of an accessible channel, and even more specifically only for using it when computing requested_by_operator.